### PR TITLE
Promote/graduate CSIMigrationAWS to GA

### PR DESCRIPTION
**Pull request by wongma7**
k/k PR: https://github.com/kubernetes/kubernetes/pull/111479
k/enhancements PR: https://github.com/kubernetes/enhancements/issues/1487

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
Thanks @wongma7 

You will also need to update https://kubernetes.io/docs/concepts/storage/volumes/#awselasticblockstore as part of documenting this change.

**Comment by wongma7** [comment]
updated [content/en/docs/concepts/storage/volumes.md](https://github.com/kubernetes/website/pull/35615/commits/4327026fef6dbe089c3f2066bb7f96c665c3b39d#diff-70b35b750a602b2e989d50acffc27244d0938f07178937655307f45acb74dbf4) !

File: content/en/docs/concepts/storage/volumes.md [hunk 1]
```diff
@@ -121,18 +121,18 @@ If the EBS volume is partitioned, you can supply the optional field `partition:
...
-{{< feature-state for_k8s_version="v1.17" state="beta" >}}
+{{< feature-state for_k8s_version="v1.25" state="stable" >}}
 
 The `CSIMigration` feature for `awsElasticBlockStore`, when enabled, redirects
 all plugin operations from the existing in-tree plugin to the `ebs.csi.aws.com` Container
 Storage Interface (CSI) driver. In order to use this feature, the [AWS EBS CSI
 driver](https://github.com/kubernetes-sigs/aws-ebs-csi-driver)
 must be installed on the cluster and the `CSIMigration` and `CSIMigrationAWS`
-beta features must be enabled.
+features must be enabled.
```
Comment (sftim) [inline content/en/docs/concepts/storage/volumes.md:131]:
```suggestion
must be installed on the cluster and the `CSIMigration` feature must be enabled.
```
AIUI, you can't disable `CSIMigrationAWS` any more.

Comment (wongma7) [reply content/en/docs/concepts/storage/volumes.md:131]: you are right, CSIMigration is also locked to true in 1.25 https://github.com/kubernetes/kubernetes/blob/master/pkg/features/kube_features.go#L884, lemme just remove the 'and the x feature must be enabled'

**Comment by wongma7** [comment]
BTW, i opened  "Require EBS CSI plugin to be installed when CSIMigrationAWS is enabled, explain why "https://github.com/kubernetes/website/pull/35647 separately because it is a bit more opinionated  and I want to keep this PR strictly about the graduation, but appreciate any reviews on that after this merges .

**Comment by krol3** [comment]
Hi @wongma7, Here Docs Shadow, please target the dev-1.25 branch
[Here's an example from GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request)

**Comment by wongma7** [comment]
@krol3 fixed noiw

**Comment by Sea-n** [comment]
Remove language labels added during changing target branch.

/remove-language fr id uk zh

File: content/en/docs/concepts/storage/volumes.md [hunk 2]
```diff
@@ -121,18 +121,17 @@ If the EBS volume is partitioned, you can supply the optional field `partition:
...
 Storage Interface (CSI) driver. In order to use this feature, the [AWS EBS CSI
 driver](https://github.com/kubernetes-sigs/aws-ebs-csi-driver)
-must be installed on the cluster and the `CSIMigration` and `CSIMigrationAWS`
-beta features must be enabled.
+must be installed on the cluster.
 
 #### AWS EBS CSI migration complete
 
-{{< feature-state for_k8s_version="v1.17" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
```
Comment (tengqm) [inline content/en/docs/concepts/storage/volumes.md:134]: Alpha in 1.21? Is this accurate?

Comment (wongma7) [reply content/en/docs/concepts/storage/volumes.md:134]: yes, https://github.com/kubernetes/kubernetes/blob/759785ea147bc13945d521eaba4a6592cbc0675f/pkg/features/kube_features.go#L389-L393

not 100% sure why it said 1.17 before

**Comment by reylejano** [comment]
@wongma7 please rebase

**Comment by wongma7** [comment]
obsoleted by https://github.com/kubernetes/website/pull/35685

**Pull request closed without merging**